wifiphisher/common/phishinghttp.py

```diff
@@ -36,38 +48,30 @@ def get(self, url):
         except:
             pass
 
-
-class ReceiveCredsHandler(tornado.web.RequestHandler):
-
-    def post(self):
-        self.redirect("/loading")
-        wifi_webserver_tmp = "/tmp/wifiphisher-webserver.tmp"
-        with open(wifi_webserver_tmp, "a+") as log_file:
-            log_file.write('[' + T + '*' + W + '] ' + O + "POST " +
-                           T + self.request.remote_ip + " " +
-                           R + repr(self.request.body) +
-                           W + "\n"
-                           )
-            log_file.close()
+    def post(self, url):
+        formData = self.request.body.split('&')
```

Comment:
The `lower_case_with_underscores ` naming convention is used everywhere in the `wifiphisher` and in order to be consistent `formData` needs to change to `form_data`.